node/core/av-store/src/lib.rs

```diff
@@ -56,22 +56,36 @@ mod columns {
 
 #[derive(Debug, Error)]
 enum Error {
-	#[error(transparent)]
-	RuntimeAPI(#[from] RuntimeApiError),
 	#[error(transparent)]
 	ChainAPI(#[from] ChainApiError),
 	#[error(transparent)]
 	Erasure(#[from] erasure::Error),
 	#[error(transparent)]
 	Io(#[from] io::Error),
 	#[error(transparent)]
-	Oneshot(#[from] oneshot::Canceled),
+	ChainAPIChannelIsClosed(#[from] oneshot::Canceled),
```

Comment:
Nit: It's `ChainApi`, [accronyms count as one word](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html).